src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,26 @@
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  error: ErrorType | null
+  setError: (error: ErrorType | null) => void
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error, setError }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
+    >
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Try to write the code in such a way that it is not necessary to disable the linter